src/app/loadout/ModPicker.tsx

```diff
@@ -261,3 +228,62 @@ function ModPicker({ plugSets, lockedMods, initialQuery, minHeight, onAccept, on
 }
 
 export default connect<StoreProps>(mapStateToProps)(ModPicker);
+
+/**
+ * Determine whether an armor mod can still be selected, given that the `selected` mods have already been selected.
+ * This doesn't take into account the actual armor that's in the loadout and what slots it has.
+ */
+function isModSelectable(
```

Comment:
This function had no dependency on any props of the component, so it could just be pulled out to a global function.

Reply:
Nice, it was using props at one stage but I must have broken that dependency recently and not put two and two together when the dep array was empty.